configure-buildfarm/templates/podtemplate-config.j2

```diff
@@ -0,0 +1,95 @@
+/**
+* Pod Template Config
+*/
+
+import org.csanchez.jenkins.plugins.kubernetes.*
+import org.csanchez.jenkins.plugins.kubernetes.volumes.*
+import jenkins.model.*
+import java.util.ArrayList
+import java.util.List
+
+def NAME="android"
+def IMAGE="{{ android_slave_image }}"
+def ENVAR_NAME="ANDROID_HOME"
+def ENVAR_VALUE="/opt/android-sdk-linux"
```

Comment:
Could we rename these ENUM names to ANDROID_HOME_NAME and ANDROID_HOME_VALUE or similar.